docs/development_suite/deploy/overview.md

```diff
@@ -178,10 +178,10 @@ Such functionalities are available through the respective buttons in the  `Confi
 #### Compare Changes
 
 :::info
-This feature is only available to Projects that use [**Enhanced Project Workflow**](/development_suite/set-up-infrastructure/enhanced-project-workflow.md) and `GitLab`, `GitHub` or `Azure` as CI/CD provider.
+This feature is only available to Projects that use [**Enhanced Project Workflow**](/development_suite/set-up-infrastructure/overview.md) and `GitLab`, `GitHub` or `Azure` as CI/CD provider.
 :::
 
-By clicking on the `Compare changes` button, a diff-editor will show the differences between the configurations of the selected version and the previous one in a side-by-side view.
+By clicking on the `Compare changes` button, a diff-editor will show the differences between the k8s configurations of the selected revision or version and the k8s files already deployed in a side-by-side view.
```

Comment:
just because i don't really like to repeat the same words in the same sentence :eyes: 

```suggestion
By clicking on the `Compare changes` button, a diff-editor will show the differences between the k8s configurations of the selected revision or version and the already deployed ones in a side-by-side view.
```